sdk/storage_queues/src/responses/get_queue_service_properties_response.rs

```diff
@@ -12,15 +13,16 @@ pub struct GetQueueServicePropertiesResponse {
 }
 
 impl std::convert::TryFrom<&Response<Bytes>> for GetQueueServicePropertiesResponse {
-    type Error = crate::Error;
+    type Error = Error;
 
-    fn try_from(response: &Response<Bytes>) -> Result<Self, Self::Error> {
+    fn try_from(response: &Response<Bytes>) -> Result<Self> {
         let headers = response.headers();
         let body = response.body();
 
         debug!("headers == {:?}", headers);
         debug!("body == {:#?}", body);
-        let queue_service_properties: QueueServiceProperties = read_xml(body)?;
+        let queue_service_properties: QueueServiceProperties =
+            read_xml(body).map_kind(ErrorKind::DataConversion)?;
```

Comment:
I imagine that  `.map_kind` will not be needed here when `read_xml` is migrated.

Reply:
I did a similar thing for the clients that are also defined by the `storage` crate.